web/templates/pages/task_edit.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'base.html' %}
+{% load staticfiles %}
+
+{% block custom_css %}
+	<link rel="stylesheet" type="text/css" href="{% static 'css/jquery.datetimepicker.css' %}" >
```

Comment:
You will need the datetimepicker files enclosed when you define the appropriate id and class using widgets in your form definition. But for now you can leave here.